### PR TITLE
PLAT-10564: Implemented UserJoinedRoom Acitivity

### PR DESCRIPTION
**Pull request by reedvfeldman**
### Ticket
[PLAT-10564](https://perzoinc.atlassian.net/browse/PLAT-10564)

### Description
Please put here the intent of your pull request.

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [ ] Referenced a ticket in the PR title and in the corresponding section
- [ ] Filled properly the description and dependencies, if any
- [ ] Unit tests updated or added
- [ ] Docstrings added or updated
- [ ] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

File: examples/activities/form_activity.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 
 async def run():
-    async with SymphonyBdk(BdkConfigLoader.load_from_symphony_dir("config.yaml")) as bdk:
+    async with SymphonyBdk(BdkConfigLoader.load_from_symphony_dir("/Users/reed.feldman/devx/test-bot-python/resources/config.yaml")) as bdk:
```
Comment (symphony-youri) [inline examples/activities/form_activity.py]: we probably don't want to push this

File: examples/activities/user_joined_room_activity.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import logging.config
+from pathlib import Path
+
+from symphony.bdk.core.activity.user_joined_room import UserJoinedRoomActivity, UserJoinedRoomContext
+from symphony.bdk.core.config.loader import BdkConfigLoader
+from symphony.bdk.core.service.message.message_service import MessageService
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+
+async def run():
+    config = BdkConfigLoader.load_from_symphony_dir("/Users/reed.feldman/devx/test-bot-python/resources/config.yaml")
```
Comment (symphony-youri) [inline examples/activities/user_joined_room_activity.py]: usually the examples use the symphony dir (i.e ~/.symphony) so we don't have to change the path to configuration

**Review by symphony-youri — COMMENTED** [review]
is that normal that the python file for the new activity is empty?

File: examples/activities/form_activity.py (hunk 1 above)
Comment (reedvfeldman) [reply examples/activities/form_activity.py]: let me revise the path to default .

File: examples/activities/user_joined_room_activity.py (hunk 2 above)
Comment (reedvfeldman) [reply examples/activities/user_joined_room_activity.py]: Let me revise the path to the default.

**Comment by symphony-elias** [comment]
Please close your previous PR, add a meaningful title and put the jira issue.
Some unit tests are missing, too.

**Comment by symphony-youri** [comment]
To add tests for the change, you can probably take existing ones from https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/tree/2.0/tests/core/activity as examples

File: tests/core/activity/registry_test.py [hunk 3]
```diff
@@ -28,6 +31,11 @@ def fixture_form():
     return MagicMock(FormReplyActivity)
 
 
+@pytest.fixture(name="user")
```
Comment (symphony-elias) [inline tests/core/activity/registry_test.py:34]: `userJoined` would be more explanatory IMHO

**Review by symphony-elias — APPROVED** [review]
Looks good !

File: tests/core/activity/registry_test.py (hunk 3 above)
Comment (symphony-youness) [reply tests/core/activity/registry_test.py:34]: +1, maybe `user_joined` instead of camel case

**Pull request merged**